packages/components/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import map from "lodash/map";
+import isNaN from "lodash/isNaN";
+import { Box } from "../../primitives/Box";
+import { Button } from "../Button";
+import { PaginationButton } from "./PaginationButton";
+import { usePagination } from "./usePagination";
+
+export interface PaginationProps
+  extends Omit<React.HtmlHTMLAttributes<HTMLDivElement>, "color"> {
+  /** The accessible label assigned to the pagination. */
+  // Label: string;
```

Comment:
Was this meant to get implemented or deleted? Having a way to set the label makes sense to me.

Reply:
added it back in